packages/location-management/src/LocationGroupModule.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Route, Switch } from 'react-router';
+import { BrowserRouter } from 'react-router-dom';
+
+import { URL_LOCATION_GROUP_ADD, URL_LOCATION_GROUP_EDIT } from './constants';
+import LocationGroupView from './components/LocationGroupView';
+import LocationGroupAdd from './components/LocationGroupAdd';
+
+/**
+ * Location Group Module
+ *
+ * @returns {Element} - Location Group Module
+ */
+export const LocationGroupModule: React.FC = () => {
+  return (
```

Comment:
Do we still need this?

Reply:
@KipSigei  i am not sure but we do plan to use nested routing in future thats why i left it in pr #74 as it is

Reply:
Not exactly sure,
I just updated the location tags
may be @ashfahan can guide more about this.

Reply:
Okay can we instead create an issue to track the nested routing issue.